packages/core/src/combinator/zipItems.js

```diff
@@ -9,7 +10,7 @@ export const withItems = (items, stream) =>
 export const zipItems = (f, items, stream) =>
   items.length === 0 || stream === empty()
```

Comment:
If we merge #151 first, then we can use `isCanonicalEmpty` here.

Reply:
💯

Reply:
... which can be done now.